controllers/front/CartController.php

```diff
@@ -63,6 +63,27 @@ public function init()
         $this->customization_id = (int)Tools::getValue('id_customization');
         $this->qty = abs(Tools::getValue('qty', 1));
         $this->id_address_delivery = (int)Tools::getValue('id_address_delivery');
+
+        if (Tools::getIsset('checkout')) {
+            $product = $this->context->cart->checkQuantities(true);
+            $isValidToCheckout = true;
+
+            if (is_array($product)) {
+                $isValidToCheckout = false;
+                $this->errors[] = sprintf(
+                    $this->trans(
```

Comment:
Do not use `sprintf()` with ` trans()`. This second function is already able to replace your variables (See https://symfony.com/doc/2.8/components/translation/usage.html#message-placeholders).

Rename `%1s` with a named variable, vove `$product['name']` in the second parameter array and you're good to remove the `sprintf()` call.